### ISSUE TITLE
Rubrics: add filters by rubric item

### ISSUE THREAD

**Issue by jonatanschroeder** [PrairieLearn/PrairieLearn#6760]
Adding this as a separate item to be considered after #6271 is done. The manual grading interface should have options to filter based on rubric items, i.e., instances where a particular rubric item has (or has not?) been applied.

**Comment by jonatanschroeder** [comment]
The grader interface could link a particular item to the search of all questions where that item was selected.

**Comment by nwalters512** [comment]
Filtering by rubric item was implemented in #12507. The "rubric item links to search" bit doesn't yet exist, but if that's really desirable, it should now be its own issue.
